### DISCUSSION TITLE
Adding a Transfer Packet?

### DISCUSSION THREAD

**Discussion by TheJavaHacker** [Minestom/Minestom#639] [Help] [answered]
Hello all,

Minecraft Bedrock Edition has a Transfer packet and many Java servers are asking for it too, and have been for several years.
That being said, do you guys reckon it's possible to add such a feature into Minestom, given that it's pretty bare-bones, meaning very customisable?

Obviously it wouldn't work out of box, it'd need a client modification to work properly, but as far as server-side implementation goes, it would be great to add it here.

Many thanks,
TheJavaHacker.

**Comment by KrystilizeNevaDies** [comment, marked as answer]
Adding a custom packet to minestom should be pretty easy, just implement [this](https://github.com/Minestom/Minestom/blob/master/src/main/java/net/minestom/server/network/packet/server/ServerPacket.java) and it should work with all our systems.

As for implementing it into minestom, it's not likely as there isn't some unified format for it. Especially considering how easy it is to do yourself.
